src/include/security/widgets.rb

```diff
@@ -261,6 +247,22 @@ def initialize_security_widgets(include_target)
             "Value"  => "100"
           }
         }
+
+      @WIDGETS.merge!(
+        @display_manager.shutdown_var_name => shutdown_login_manager_widget
+      ) if @display_manager
```

Comment:
Do not use trailing `if` for multiline command. (We should enable rubocop for this module... :wink:)